composer.json

```diff
@@ -0,0 +1,27 @@
+{
+    "name": "fourkitchens/gatsby-starter-emulsify-drupal",
+    "description": "Gatsby Starter using the Emulsify Gatsby Theme for a Design System, Storybook for a Pattern Library and serves as a starterkit Drupal 8 theme.",
+    "type": "drupal-theme",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Evan Willhite",
+            "email": "evan@fourkitchens.com"
+        }
+    ],
+    "repositories": {
+        "drupal": {
+            "type": "composer",
+            "url": "https://packages.drupal.org/8"
+        }
+    },
+    "minimum-stability": "dev",
+    "prefer-stable": true,
+    "require": {
+        "drupal/components": "^1.0",
+        "drupal-pattern-lab/add-attributes-twig-extension": "^1.0",
```

Comment:
Should we consider forking these `drupal-pattern-lab` paths into Four Kitchens org or creating an emulsify organization to house these so that we don't have confusion of this using pattern lab?

Reply:
These are fine - this is a shared repo with some other developers so the name would have to be changed between us all.